lib/middleware.js

```diff
@@ -132,8 +132,11 @@ exports.checkQuota = function () {
     if (utils.isServiceFabrikOperation(req.body)) {
       logger.debug('[Quota]: ServiceFabrikOperation : calling next handler..');
       next();
+    } else if (req.body.context.platform !== 'cloudfoundry') {
```

Comment:
Same as [above](https://github.com/cloudfoundry-incubator/service-fabrik-broker/pull/131/files#r157121565)

Reply:
This is the only place where in middleware we have used space_guid and org_guid. As stated above, other checks will go off with SF2.0. Hence I thought we can keep this check.